apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html

```diff
@@ -96,8 +107,10 @@ <h5>
                             {{localeData?.unused_receipts}}
                             <div>
                                 <span class="d-inline-flex number-total">
-                                    {{receiptsSummaryData?.unUsed?.symbol}}{{receiptsSummaryData?.unUsed?.amount |
-                                    giddhCurrency}}/-
+                                    <!-- {{receiptsSummaryData?.unUsed?.symbol}}{{receiptsSummaryData?.unUsed?.amount |
+                                    giddhCurrency}}/- -->
+                                    <amount-field [amount]="receiptsSummaryData?.unUsed?.amount" [currencySymbol]="receiptsSummaryData?.unUsed?.symbol" [currencyCode]="false">
```

Comment:
Do check currencyCode is not passed but currencySymbol is provided which will not make it RTL for Gulf countries.

Reply:
Done